src/System.Console/src/System.Console.csproj

```diff
@@ -93,6 +93,7 @@
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
     <Compile Include="System\ConsolePal.Unix.cs" />
+    <Compile Include="System\InternalsVisibleTo.Unix.cs" />
```

Comment:
This is looking for InternalsVisibleTo.Unix.cs, but the file is named InternalsVisibleTo.cs.  I'd leave it as InternalsVisibleTo.cs and update this .csproj line accordingly.